packages/cli/__tests__/zosfiles/__unit__/create/like/ds.handler.unit.test.ts

```diff
@@ -0,0 +1,100 @@
+            // Require the handler and create a new instance
+            const handlerReq = require("../../../../../src/zosfiles/create/classicPds/ds.handler");
+            const handler = new handlerReq.default();
+            const dataSetName = "testing";
+
+            // Vars populated by the mocked function
+            let error;
+            let apiMessage = "";
+            let jsonObj;
+            let logMessage = "";
+            let fakeSession = null;
+
+            // Mock the create function
+            Create.dataSet = jest.fn((session) => {
+                fakeSession = session;
+                return {
+                    success: true,
+                    commandResponse: "created"
+                };
+            });
+
+            // Mocked function references
+            const profFunc = jest.fn((args) => {
+                return {
+                    host: "fake",
+                    port: "fake",
+                    user: "fake",
+                    password: "fake",
+                    auth: "fake",
+                    rejectUnauthorized: "fake"
+                };
+            });
+
+            try {
+                // Invoke the handler with a full set of mocked arguments and response functions
+                await handler.process({
+                    arguments: {
+                        $0: "fake",
+                        _: ["fake"],
+                        dataSetName,
+                        ...UNIT_TEST_ZOSMF_PROF_OPTS
+                    },
+                    response: {
+                        data: {
+                            setMessage: jest.fn((setMsgArgs) => {
+                                apiMessage = setMsgArgs;
+                            }),
+                            setObj: jest.fn((setObjArgs) => {
+                                jsonObj = setObjArgs;
+                            })
+                        },
+                        console: {
+                            log: jest.fn((logArgs) => {
+                                logMessage += "\n" + logArgs;
+                            })
+                        },
+                        progress: {
+                            startBar: jest.fn((parms) => {
+                                // do nothing
+                            }),
+                            endBar: jest.fn(() => {
+                                // do nothing
+                            })
+                        }
+                    },
+                    profiles: {
+                        get: profFunc
+                    }
+                } as any);
+            } catch (e) {
+                error = e;
+            }
+
+            expect(error).toBeUndefined();
+            expect(profFunc).toHaveBeenCalledWith("zosmf", false);
+            expect(Create.dataSet).toHaveBeenCalledTimes(1);
+            expect(Create.dataSet).toHaveBeenCalledWith(fakeSession, CreateDataSetTypeEnum.DATA_SET_CLASSIC, dataSetName, {});
```

Comment:
DATA_SET_CLASSIC should be DATA_SET_LIKE here

Reply:
DATA_SET_LIKE does not exist in createdatasettypeenum

Reply:
Sorry about that 😢 In this case you'll want to test that `Create.dataSetLike` was called rather than `Create.dataSet`. Please also update the expected parameters to match `Create.dataSetLike(session: AbstractSession, dataSetName: string, likeDataSetName: string)`.